webapp/channels/src/components/suggestion/search_channel_with_permissions_provider.test.tsx

```diff
@@ -1,12 +1,12 @@
 // Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
 // See LICENSE.txt for license information.
 
-import {getState} from 'stores/redux_store';
+import SearchChannelWithPermissionsProvider from 'components/suggestion/search_channel_with_permissions_provider';
 
+import store from 'stores/redux_store';
+const getState = store.getState;
```

Comment:
can we keep the variable declaration separate from import :)

Reply:
done